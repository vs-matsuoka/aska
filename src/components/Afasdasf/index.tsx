const Afasdasf = () => (
  <h1>Afasdasf</h1>
)

export default Afasdasf
