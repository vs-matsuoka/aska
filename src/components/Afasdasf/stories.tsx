import { Story, Meta } from '@storybook/react'
import Afasdasf from '.'

export default {
  title: 'Afasdasf',
  component: Afasdasf
} as Meta

export const Default: Story = () => <Afasdasf />
