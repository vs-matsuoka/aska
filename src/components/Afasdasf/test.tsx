import { render, screen } from '@testing-library/react'

import Afasdasf from '.'

describe('<Afasdasf />', () => {
  it('should render the heading', () => {
    const { container } = render(<Afasdasf />)

    expect(screen.getByRole('heading', { name: /Afasdasf/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
