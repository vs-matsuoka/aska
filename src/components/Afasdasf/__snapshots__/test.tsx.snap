// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Afasdasf /> should render the heading 1`] = `
<h1>
  Afasdasf
</h1>
`;
